frb_dart/lib/src/ffi/stub.dart

```diff
@@ -19,6 +20,19 @@ abstract class FlutterRustBridgeWireBase {
   // ignore: non_constant_identifier_names
   void store_dart_post_cobject(DartPostCObject ptr) {}
 
+  // ignore: non_constant_identifier_names
+  Object get_dart_object(int ptr) {
+    return '';
```

Comment:
Given that this base class is abstract, shall we use `Object get_dart_object();` i.e. no body at all

same for all other methods

Reply:
it's because of dart analyzer. he does not see that we are importing web.dart from the implementation and consider that there are not enough symbols.

Reply:
Weird. Could you please take a screenshot of its complain?

Reply:
https://github.com/fzyzcjy/flutter_rust_bridge/actions/runs/3600007687/jobs/6064198006
Analyzing dart...

  error - lib/bridge_generated.web.dart:112[8](https://github.com/rogurotus/flutter_rust_bridge/actions/runs/3600007442/jobs/6064197497#step:9:9):7 - Missing concrete implementations of 'FlutterRustBridgeWireBase.drop_dart_object', 'FlutterRustBridgeWireBase.free_WireSyncReturnStruct', 'FlutterRustBridgeWireBase.get_dart_object', 'FlutterRustBridgeWireBase.new_dart_opaque', and 1 more. Try implementing the missing methods, or make the class abstract. - non_abstract_class_inherits_abstract_member

Reply:
the analyzer still responds the same way

Reply:
Hmm looks like the problem is, in web.dart, FlutterRustBridgeExampleSingleBlockTestWire really does not implement those methods.

Thus, seems that:

1. FlutterRustBridgeWasmWireBase should implement those method (by throwing unimplemented error etc if you think it is not needed)
2. FlutterRustBridgeWireBase should be abstract methods

![image](https://user-images.githubusercontent.com/5236035/205410283-c576cba9-e2fa-4492-8f14-54eca55e687b.png)

Reply:
the implementation from web.dart is used.

this is due to the fact that native uses the generated version, and web uses the implementation from web.dart

Reply:
I see. Without making it more complicated, this LGTM